automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspacesAzureApiSpec.scala

```diff
@@ -157,12 +157,16 @@ class WorkspacesAzureApiSpec extends AnyFlatSpec with Matchers with BeforeAndAft
           verifyBlobNotCloned(cloneSasUrl, nonAnalysesFilename)
         }
       } finally {
-        Rawls.workspaces.delete(projectName, workspaceCloneName)
-        assertNoAccessToWorkspace(projectName, workspaceCloneName)
+        withClue(s"deleting the cloned workspace ${workspaceCloneName} failed") {
```

Comment:
In one of the iterations, I got a failure from deleting one of these workspaces, and it was unclear to me which one it was.